documentation/internal/Release.md

```diff
@@ -0,0 +1,15 @@
+# Releasing KubeMarine
+
+If you want to make a new KubeMarine release, you need to do following:
+1. On the target git branch, update KubeMarine version and create tag (replace `X.X.X` with actual version):
```

Comment:
Need to mention to use main only as a target branch

Reply:
Specified `main` branch explicitly